roles/postgresql/tasks/public_repo.yml

```diff
@@ -2,6 +2,13 @@
 # SPDX-License-Identifier: Apache-2.0
 
 ---
+- name: Install latest version of ca-certificates
+  yum:
+    name: ca-certificates
```

Comment:
I do not think we should do the update. This is related to have up to date security packages which is not the scope of TDP.

Reply:
Agreed, it should be left to the sys admins